README.md

```diff
@@ -125,11 +121,14 @@ When a task starts running, it starts a pod and runs each step sequentially in a
 
 Note that only the requirement for a git repository is declared on the task and not a specific git repository to be used. That allows tasks to be reusable for multiple pipelines and purposes. You can find more examples of reusable tasks in the [Tekton Catalog](https://github.com/tektoncd/catalog) and [OpenShift Catalog](https://github.com/openshift/pipelines-catalog) repositories.
 
-Install the `openshift-client` and `s2i-java` tasks from the catalog repository using `oc` or `kubectl`, which you will need for creating a pipeline in the next section:
+Install the `apply-manifests` and `update-deployment` tasks from the repository, `buildah` and `s2i-python-3` tasks from catalog repository
+using `oc` or `kubectl`, which you will need for creating a pipeline in the next section:
 
 ```bash
-$ oc create -f https://raw.githubusercontent.com/openshift/tektoncd-catalog/release-v0.7/openshift-client/openshift-client-task.yaml
-$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-catalog/release-v0.7/s2i-java-8/s2i-java-8-task.yaml
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/voting-pipeline/update_deployment_task.yaml
```

Comment:
can't we use the `openshift-cli` task instead of `update_deployment_task` and `apply_manifest_task`?

Reply:
no, we need to pass the resource to apply the manifests from individual repo's